index.d.ts

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+export interface BusyIndicatorProperties {
+  color: string;
+  overlayColor: string;
+  overlayHeight: number;
+  overlayWidth: number;
+  size: 'large' | 'small';
+  startVisible: boolean;
+  text: string;
+  textColor: string;
+  textFontSize: number;
+  textNumberOfLines: number;
+}
+
+export interface BusyIndicatorState {
```

Comment:
Could we rename this something else to avoid confusion with _actual_ component state, as this isnt the entire state.

Could you also add another interface that contains the actual component state (missing `title`), and it extend the above interface to make things even cleaner

Reply:
I can name it `BusyIndicatorDefaultState` ?

Reply:
That still feels wrong, as it's not the default state. I'd suggest `BusyIndicatorState` and `BusyIndicatorEffect` perhaps?

Reply:
What do BusyIndicatorEffect stand for?